src/ethereum/utils.py

```diff
@@ -34,3 +35,24 @@ def get_sign(value: int) -> int:
         return 0
     else:
         return 1
+
+
+def ceil32(value: U256) -> U256:
+    """
+    Converts a 256-bit unsigned integer to the next closest multiple of 32.
+
+    value :
+        The value whose ceil32 is to be calculated.
+
+    Returns
+    -------
+    ceil32 : `ethereum.base_types.U256`
+        The same value if its a perfect multiple of 32
+        else it gets the next closest multiple of 32.
```

Comment:
Maybe the one place a slightly more mathematical explanation is better :stuck_out_tongue_closed_eyes: 

```suggestion
        else it returns the smallest multiple of 32 that is greater than `value`.
```

Reply:
Thanks for helping me fix the docs. Docs is the part I struggle with the most being a non-native English speaker.